board/raspberrypi/rpi/rpi.c

```diff
@@ -588,3 +588,19 @@ int ft_board_setup(void *blob, struct bd_info *bd)
 
 	return 0;
 }
+
```

Comment:
I am not sure if this is a HACK. I saw exactly the same code for other platforms. I assume this is normal for U-BOOT

Reply:
I agree that this may be proper solution for u-boot. I was thinking about rp1 driver itself. I suppose it should be moved out from this call once we get is included into the pcie device chain.

Reply:
Yep. It's probably not a HACK.

Reply:
Ok. I will remove HACK tag but will leave a NOTE with a few words if you don't mind

Reply:
removed